Hashing/C++/9_Distinct Echo Substrings (HASHING) .cpp

```diff
@@ -0,0 +1,47 @@
+//brute force approach
+//time complexity=0(n2)
+//space complexity 0(n)
+
+class Solution {
```

Comment:
You are using class but not declaring class in main function. Either declare remove the class or declare class and obj in main. @soma2000-lang